packages/framework/presence/src/index.ts

```diff
@@ -37,11 +37,11 @@ export type {
 	PresenceWorkspaceAddress,
 } from "./types.js";
 
-export type {
-	ClientSessionId,
-	IPresence,
-	ISessionClient,
-	PresenceEvents,
+export {
+	type ClientSessionId,
+	type IPresence,
+	type ISessionClient,
+	type PresenceEvents,
 	SessionClientStatus,
 } from "./presence.js";
```

Comment:
Use the two import/export pattern. One for type and one to values.
API reports don't change?

Reply:
Nope, still shows the type and the const exported

Reply:
@Josmithr, looks like we found an api-extractor bug. If a type/value pair exist but only the type is exported, then api-extractor will report the value is exported too. Sound like a known issue?